src/en/credentials.md

```diff
@@ -13,57 +13,51 @@ We use the term *credentials* to describe the material necessary to do this
 represented by a *credential name* that is used to refer to those credentials
 in subsequent commands.
 
-Juju selects a credential according to how many credentials are defined. If you
-have only one credential, or if a credential is labelled 'default', then this
-is the credential that will be used by Juju. When multiple credentials are
-defined, with no default, a credential name must be specified at the model
-level.
-
+When credentials are added for a given cloud they become available to use on
+that cloud's controller and models. There are therefore two categories of
+credentials: those that are available (local) and those that have been uploaded
+to a controller (remote) due to requiring a credential for the first time.
+
+An active credential (one that has been used to bring about a change in Juju)
+is always associated with a cloud, a Juju user, and a model.
+ 
 ## Adding credentials
 
 Juju supports three methods for adding credentials:
 
- - Accepting credentials provided interactively by the user on the command line
- - Scanning for existing credentials via environment variables and/or "rc"
+ - accepting credentials provided interactively by the user on the command line
+ - scanning for existing credentials via environment variables and/or "rc"
    files (only supported by certain providers)
- - Reading a user-provided [YAML-formatted][yaml] file
+ - reading a user-provided [YAML-formatted][yaml] file
   
 !!! Note:
-    LXD deployments are a special case. Accessed from a Juju admin user, they
-    do not require credentials. Accessed from a non-admin user, a *certificate
-    credential* is needed. See
+    Local LXD deployments are a special case. Accessed from a Juju admin user,
+    they do not require credentials. Accessed from a non-admin user, a
```

Comment:
This isn't true is it, even local LXD deployments require credentials, the "localhost" credentials are stored inside the users local `credentials.yaml`. In fact they have an expiry upon them (ten years from initial creation).

Reply:
Right. Here I'm letting the user know they don't need to *add* a credential. I'll reword a bit.

Reply:
Agreed, but the user isn't required to setup those credentials, unlike remote LXD and other clouds.

Reply:
No, but they should be maybe aware there is an expiry?

Reply:
I added the expiry.